openstack/blockstorage/v1/apiversions/urls.go

```diff
@@ -11,5 +12,7 @@ func getURL(c *gophercloud.ServiceClient, version string) string {
 }
 
 func listURL(c *gophercloud.ServiceClient) string {
-	return c.ServiceURL("")
+	u, _ := url.Parse(c.ServiceURL(""))
+	u.Path = "/"
+	return u.String()
```

Comment:
Problem is ```c.Endpoint``` contains full service URL with path included
like ```http://public.fuel.local:8776/v2/d98e06adf5db49ad9f372625cad7840b/``` 
while for list of avaiable API version one needs to call root path 
like ```http://public.fuel.local:8776/```